src/components/CallView/Grid/Grid.vue

```diff
@@ -828,6 +922,82 @@ export default {
 			return callParticipantModel.attributes.peerId === this.$store.getters.selectedVideoPeerId
 		},
 
+		isModelWithVideo(callParticipantModel) {
+			return callParticipantModel.attributes.videoAvailable
+				&& this.sharedDatas[callParticipantModel.attributes.peerId].remoteVideoBlocker.isVideoEnabled()
```

Comment:
That re-sorts the list, if user-receiver disables video of other participant. I think, only ones that disabled by sender should be falsy